Library/Formula/odb.rb

```diff
@@ -0,0 +1,60 @@
+class Odb < Formula
+  homepage "http://www.codesynthesis.com/products/odb/"
+  url "http://www.codesynthesis.com/download/odb/2.4/odb-2.4.0.tar.gz"
+  sha256 "169103a7829b9d8b2fdf5c267d18acc3d47c964d355c7af335d75c63b29c52b5"
+
+  depends_on "gcc"
+  depends_on "libcutl"
+
+  fails_with :clang do
+    cause "ODB is a GCC plugin, therefore must be built with gcc"
+  end
+
+  fails_with :llvm do
+    cause "ODB is a GCC plugin, therefore must be built with gcc"
+  end
```

Comment:
As a reminder: `odb` is a `gcc` plugin, it needs to be compiled with it. Apparently, `depends_on` `gcc` doesn't build the formula with `gcc`. So I needed to add `fails_with`